Formula/gemmi.rb

```diff
@@ -16,6 +16,7 @@ class Gemmi < Formula
 
   depends_on "cmake" => :build
   depends_on "nanobind" => :build
+  depends_on "ninja" => :build
   depends_on "robin-map" => :build
   depends_on "python3"
```

Comment:
"python@3.13" is preferred to "python3". This is because a build using a newer version of Python may fail in the future.